rest_api/application.py

```diff
@@ -21,11 +20,6 @@ def get_application() -> FastAPI:
         CORSMiddleware, allow_origins=["*"], allow_credentials=True, allow_methods=["*"], allow_headers=["*"],
```

Comment:
We can add one line comment about security implications (https://fastapi.tiangolo.com/tutorial/cors/). As `allow_origins=["*"]` is bad for security purpose.